README.md

```diff
@@ -2,10 +2,62 @@
 
 DeltaCAT is a Pythonic Data Catalog powered by Ray.
 
-Its data storage model allows you to define and manage fast, scalable, 
-ACID-compliant data catalogs through git-like stage/commit APIs, and has been 
+Its data storage model allows you to define and manage fast, scalable,
+ACID-compliant data catalogs through git-like stage/commit APIs, and has been
 used to successfully host exabyte-scale enterprise data lakes.
 
 DeltaCAT uses the Ray distributed compute framework together with Apache Arrow
-for common table management tasks, including petabyte-scale 
+for common table management tasks, including petabyte-scale
 change-data-capture, data consistency checks, and table repair.
+
+## Getting Started
```

Comment:
a readme file for a public lib should aim for non-developers; development section can be marked as How to contribute, etc. take a look of other oss projects to get some template.

Reply:
Updated the README